tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -114,82 +370,608 @@ To see a list of image families:
+
+    ```sh
+    gcloud compute firewall-rules create http --allow=tcp:80 \
+    --target-tags=http-server --source-ranges=0.0.0.0/0
+    ```
 
-## Create a compute instance for the image in your gcp project
+2. Create an instance using the newly created Linux image
 
-Once the build completes, create the instance and requisite firewall rule to test that the image works.
+    ```sh
+    gcloud compute instances create helloworld-from-factory \
+    --image https://www.googleapis.com/compute/v1/projects/$PROJECT/global/images/helloworld-v01 \
+    --tags=http-server --zone=$ZONE
+    ```
 
-    gcloud compute firewall-rules create http --allow=tcp:80 --target-tags=http-server --source-ranges=0.0.0.0/0
-    gcloud compute instances create helloworld-from-factory --image https://www.googleapis.com/compute/v1/projects/$PROJECT/global/images/helloworld-v01 --tags=http-server --zone=$ZONE
+</details>
 
-## Check the website
+## Building a Windows Image
 
-Wait a minute or two and open the browser to the IP address of the instance to see the special message.
+<details>
+<summary>Click to reveal commands</summary>
 
-1.  Retrieve the instace IP address:
+1. Open the [**Compute Engine** page](https://console.cloud.google.com/compute) in the Cloud Console and 
+navigate to **Images** to see the Packer generated instance image.
 
-        gcloud compute instances list --filter="name:helloworld*" --format="value(networkInterfaces[0].accessConfigs[0].natIP)"
+2. Select the image and click on **CREATE INSTANCE**. Complete the launch wizard to launch the instance, whilst ensuring that the **Boot disk** is set to use our custom Image.
 
-2.  Go to the IP address in the browser and make sure that you see the "Hello from the image factory!" message.
+</details>
 
-## Cleaning up
+# Task 12: Verifying the results of our efforts 
+Now that we've successfully used packer to generate a new golden image and create a new instance, follow the instructions to verify that our deployment has worked correctly.
+
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+1. Wait a minute or two and open the browser to the IP address of the instance to see the special message.
+
+1. Retrieve the instace IP address:
+
+    ```sh
+    gcloud compute instances list --filter="name:helloworld*" --format="value(networkInterfaces[0].accessConfigs[0].natIP)"
+    ```
+
+1. Go to the IP address in the browser and make sure that you see the "Hello from the image factory!" message.
+
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+1. Wait a few minutes until the Windows VM has completed the boot up process.
+
+1. Using the [Chrome RDP plugin](https://cloud.google.com/compute/docs/instances/›connecting-to-instance#chrome-remote-desktop) or your preferred [Remote Desktop client](https://cloud.google.com/compute/docs/instances/connecting-to-instance#windows), RDP into your Windows instance.
```

Comment:
Link to https://cloud.google.com/compute/docs/instances/connecting-to-instance instead of suggesting a specific way to connect

Reply:
It appears that this suggestion was implemented.

Reply:
ack